### DISCUSSION TITLE
HTTP GET/POST/DELET API SECURITY

### DISCUSSION THREAD

**Discussion by kratos9008** [devlikeapro/waha#696] [Q&A]
How do I protect myself so that anyone who knows my URL cannot use my services, since when doing any POS/GET/DELETE, etc., in the HEADER they do not need an API key, despite having it configured in the dashboard?

**Comment by devlikepro** [comment]
Hi! you need to follow the guide https://waha.devlike.pro/docs/how-to/install/
Remember to change `WHATSAPP_API_KEY` in `.env` file, it's `123` by default

The [🔒 Security](https://waha.devlike.pro/docs/how-to/security/) settings works only in [**WAHA Plus**](https://waha.devlike.pro/docs/how-to/waha-plus/) right now

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
